docs/source/linchpincli_linchpin_invgen.rst

```diff
@@ -1,24 +0,0 @@
-Invgen
```

Comment:
And derp, since I have the stubs I just left them in the toctree -- no warnings are generated. I put a note in each command stub doc saying that the command hasn't been implemented yet (changes coming up shortly).